src/rules/javascript/deprecated_entities.js

```diff
@@ -0,0 +1,25 @@
+import { DEPRECATED_ENTITIES } from 'const';
+import { getNodeReferenceName } from 'utils';
+
+export function deprecated_entities(context) {
+  return {
+    CallExpression: function(node) {
+      var objectName;
+      // We're only looking for calls that look like `foo.bar()`
+      if (typeof node.callee.object !== 'undefined' &&
+          node.callee.property.type === 'Identifier' &&
+          node.callee.object.type === 'Identifier') {
+
+        objectName = getNodeReferenceName(context, node.callee.object);
+
+        for (let entity of DEPRECATED_ENTITIES) {
+          // check to see if the node is matches a deprecated entity
```

Comment:
Minor nit - all comments should be sentences starting with a capital letter and ending with a period.

Also there's a typo 'is matches'